v4/source/plugins.yaml

```
@@ -553,3 +553,50 @@
             }
 
             ok, resp = Client.GetMarketplacePlugins(filter)
+  /plugins/first_admin_visit_marketplace:
+    get:
+      tags:
+        - system
+      summary: Retrieves the system-level status that specifies that at least one System Admin has visited the in-product Plugin Marketplace.
+      description: |
+        Retrieves the status that specifies that at least one System Admin has visited the in-product Plugin Marketplace.
+        __Minimum server version: 5.33__
+        ##### Permissions
+        Users must be System Admins - have `manage_system` permissions.
```

At least in the plugin section this wording is used. Should we stick to it for consistency? 
```suggestion
        Must have `manage_system` permissions.
```